develop/tutorials/articles/192-item-selector/01-selecting-entities-using-the-item-selector-api.markdown

```diff
@@ -0,0 +1,356 @@
+# Selecting Entities Using the Item Selector API [](id=selecting-entities-using-the-item-selector-api)
```

Comment:
@sergiogonzalez - You're viewing the right tutorial ( https://github.com/mwilliams2014/liferay-docs/blob/d462cbc1241ff4c11bb9a33cfe460b2e78d81d15/develop/tutorials/articles/192-item-selector/01-selecting-entities-using-the-item-selector-api.markdown ). I'm suggesting we improve the title by expressing that the tutorial shows developers how to populate and display an item selector in their app. The user selects items, whereas the developer enables the app to display the item selector.

Here are some title options to consider:
- Providing an Item Selector (I like this one most)
- Assembling an Item Selector

Make sense? Other title suggestions?